### DISCUSSION TITLE
Turborepo deployments failing on the first try "routes-manifest.json couldn't be found"

### DISCUSSION THREAD

**Discussion by TaylorFacen** [vercel/vercel#9144] [Help]
Whenever my app is deployed, I get the error
```
Error: The file "/vercel/path0/apps/marketing/apps/marketing/.next/routes-manifest.json" couldn't be found. This is normally caused by a misconfiguration in your project.
```

What's interesting is that when I redeploy the deployment, it works. I've already verified that the "name" in my package.json matches the name of the folder. I'm using a monorepo (turborepo) with two apps: app and marketing both with the default build settings. 

Also, whenever I push some work completed in my "app" app, my "marketing" app gets redeployed even though there's no changes in my marketing folder. My "app" app deploys just fine. 

![Screen Shot 2022-12-30 at 2 21 12 AM](https://user-images.githubusercontent.com/44216029/210044876-452e0b45-89f7-442b-bb4d-8ef34b8dec89.png)

![Screen Shot 2022-12-30 at 2 22 12 AM](https://user-images.githubusercontent.com/44216029/210044954-172f2e4d-4c2f-4ed1-b057-6ca25f69b77c.png)

**Comment by michael-land** [comment]
Encountered this issue today while working on a new project. It appears to occur only when there's a cache hit, and Turbo serves the cached output. I resolved it by updating the turbo.json configuration to include:

```json
"outputs": [ ".next/**", "!.next/cache/**"]
```
in the outputs section. 

This worked for me.
